CHANGELOG.md

```diff
@@ -10,7 +10,15 @@ corresponding pull request appended.
 
 ## [Unreleased]
 
-## [2.0.0] - 2019-YY-ZZ
+## [2.1.0] - 2019-YY-ZZ
+
+### Fixed
+
+ - Allow referencing computed values. [#43]
+
+- Fix issue with long IAM bindings list. [#32]
```

Comment:
this is not belong here

Reply:
I don't think so because `- Fix issue with long IAM bindings list. [#32] ` it was my fix from previously merged pull request

Reply:
this code should not be pushed to aaron-lane:12

Reply:
Done